codegen/src/main/kotlin/software/amazon/smithy/rust/codegen/smithy/generators/client/FluentClientDecorator.kt

```diff
@@ -184,37 +184,77 @@ class GenericFluentClient(codegenContext: CodegenContext) : FluentClientCustomiz
                     // Ignored as otherwise we'd need to pull in all these dev-dependencies.
                     /// ```rust,ignore
                     /// use aws_endpoint::AwsEndpointStage;
+                    /// use aws_http::auth::CredentialsStage;
+                    /// use aws_http::recursion_detection::RecursionDetectionStage;
                     /// use aws_http::user_agent::UserAgentStage;
                     /// use aws_sig_auth::middleware::SigV4SigningStage;
                     /// use aws_sig_auth::signer::SigV4Signer;
-                    /// use aws_smithy_http_tower::map_request::MapRequestLayer;
-                    /// use tower::layer::util::Stack;
+                    /// use aws_smithy_client::retry::Config as RetryConfig;
+                    /// use aws_smithy_http_tower::map_request::{AsyncMapRequestLayer, MapRequestLayer};
+                    /// use std::fmt::Debug;
+                    /// use tower::layer::util::{Identity, Stack};
                     /// use tower::ServiceBuilder;
                     ///
-                    /// type AwsMiddlewareStack =
-                    ///     Stack<MapRequestLayer<SigV4SigningStage>,
-                    ///         Stack<MapRequestLayer<UserAgentStage>,
-                    ///             MapRequestLayer<AwsEndpointStage>>>,
+                    /// type AwsMiddlewareStack = Stack<
+                    ///     MapRequestLayer<RecursionDetectionStage>,
+                    ///     Stack<
+                    ///         MapRequestLayer<SigV4SigningStage>,
+                    ///         Stack<
+                    ///             AsyncMapRequestLayer<CredentialsStage>,
+                    ///             Stack<
+                    ///                 MapRequestLayer<UserAgentStage>,
+                    ///                 Stack<MapRequestLayer<AwsEndpointStage>, Identity>,
+                    ///             >,
+                    ///         >,
+                    ///     >,
+                    /// >;
                     ///
-                    /// ##[derive(Debug, Default)]
+                    /// /// AWS Middleware Stack
+                    /// ///
+                    /// /// This implements the middleware stack for this service. It will:
+                    /// /// 1. Load credentials asynchronously into the property bag
+                    /// /// 2. Sign the request with SigV4
+                    /// /// 3. Resolve an Endpoint for the request
+                    /// /// 4. Add a user agent to the request
+                    /// #[derive(Debug, Default, Clone)]
+                    /// #[non_exhaustive]
```

Comment:
I think CI is failing because these need to be `##` rather than `#`

Reply:
Ah, good catch!